ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -1803,6 +1843,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-gcp-ipi-proxy-techpreview
+- as: gcp-ipi-proxy-techpreview-p3-f28-long-duration-cloud
+  cron: 22 10 25 * *
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      E2E_RUN_TAGS: '@gcp-ipi and @network-ovnkubernetes and not @fips and @long-duration'
+      TAG_VERSION: '@4.12'
+      TEST_SCENARIOS: Cluster_Infrastructure
+      TEST_TIMEOUT: "90"
+    test:
+    - chain: openshift-e2e-test-qe-long-duration
+    workflow: cucushift-installer-rehearse-gcp-ipi-proxy-techpreview
```

Comment:
This one is same with 4.12, maybe we can change it to `cucushift-installer-rehearse-gcp-ipi-private`

Reply:
done